src/rtc/rtc.c

```diff
@@ -970,11 +970,8 @@ s32 sceRtcFormatRFC2822LocalTime(char *pszDateTime, const u64 *pUtc) {
     int oldK1 = pspShiftK1();
     s32 res = SCE_ERROR_INVALID_POINTER;
 
-    if((oldK1 & (int)pszDateTime) >= 0) {
-        if((oldK1 & (int)pszDateTime) >= 0) {
+    if(((oldK1 & (int)pszDateTime) >= 0) && ((oldK1 & (int)pszDateTime) >= 0))
```

Comment:
We do have special inline functions just for register K1 checking which should be used, [here](https://github.com/uofw/uofw/blob/master/include/common/registers.h#L35).